packages/validation/__tests__/url.test.js

```diff
@@ -55,4 +55,13 @@ describe("url test", () => {
             true
         );
     });
+
+    it("should pass - href URL", () => {
```

Comment:
I'm interested to know how the `allowHref` behave when a string containing white space is passed. For example,

- `mailto:abc @ gmail.com`
- `#hello   world`

IMHO we should at least also handle that URL is a continuous string without any whitespace.
What do you think? @sccalabr

Reply:
sure we can do that.